.flake8

```diff
@@ -5,7 +5,7 @@
 select = E,F,W,B,B901,B902,B903
 
 # ignore W504 line break before binary operator
-ignore = W504
```

Comment:
This is a new flake8 warning and adding it to ignore is fine for now but reading it I think we should probably make some efforts to update the test code (per the flake8 warning: be more specific than `pytest.raises(Exception)` – use the specific Exception that is expected to be raised) and remove B017 from the ignore list once that's done.

Reply:
yeah, I added this because it suddenly appeared and was causing the tests to fail. I'm fine with fixing it when we have time.